web/app/scripts/filterbar/filterbar-controller.js

```diff
@@ -54,9 +61,11 @@
         /**
          * When the record type changes, request the new schema
          */
-        $scope.$on('driver.state.recordstate:selected', function(event, selected) {
-            if (selected) {
+        $scope.$on('driver.state.recordstate:selected',
+                   function(event, selected) { onRecordSelected(selected); });
```

Comment:
Yes, but shouldn't the first argument sent by `driver.state.recordstate:selected` still be the event itself, and not `selected`? Looks like it's sending the event as a parameter named `selected` to `onRecordSelected`, instead of a second parameter from `driver.state.recordstate:selected` that should be the `selected` thing.